### PR TITLE
Added support for unified report

### PR DESCRIPTION
**Pull request by ofirtroushinsky**

### CODE REVIEW

File: singular_api_client/singular_client.py [hunk 1]
```diff
@@ -136,6 +136,49 @@ def create_async_skadnetwork_raw_report(self, start_date, end_date,
...
+                                    cohort_periods=None,
+                                    source=None,
+                                    app=None,
+                                    time_breakdown=TimeBreakdown.ALL,
+                                    country_code_format=CountryCodeFormat.ISO3,
+                                    filters=None,
+                                    **kwargs
+                                    ):
+        """
+        Use this endpoint to run custom queries in the Singular platform for aggregated unified  data without
```
Comment (yamitkaduri) [inline singular_api_client/singular_client.py]: You have extra space here: "aggregated unified  data"

File: singular_api_client/singular_client.py [hunk 2]
```diff
@@ -136,6 +136,49 @@ def create_async_skadnetwork_raw_report(self, start_date, end_date,
...
+         keeping a live connection throughout the request
+
+        :param start_date: "YYYY-mm-dd" format date
+        :param end_date: "YYYY-mm-dd" format date
+        :param format: Format for returned results, for example Format.JSON
+        :param dimensions: A list of dimensions, for example [Dimensions.APP, Dimensions.Source]
+        :param metrics: A list of metrics, for example [Metrics.UNIFIED_INSTALLS]
+        :param cohort_metrics: list of cohorted metrics by name or ID
```
Comment (yamitkaduri) [inline singular_api_client/singular_client.py]: Better to start all params description with capital letter :)

**Review by yamitkaduri — APPROVED** [review]
Really niceee, just small comments

**Comment by yonathantzur** [comment]
bump version on singular_api_client/version.py before merging to master

**Pull request merged**